src/components/[guild]/Requirements/components/GuildCheckout/components/PurchaseFeeAndTotal.tsx

```diff
@@ -69,22 +69,28 @@ const PurchaseFeeAndTotal = (): JSX.Element => {
               <Icon as={Info} color="gray" />
             </Tooltip>
           </HStack>
-          <Text as="span" colorScheme="gray">
-            {error ? (
-              "Couldn't calculate"
-            ) : pickedCurrency ? (
-              <>
-                {isTooSmallFee
-                  ? "< 0.001"
-                  : (calculatedGasFee ?? 0 + guildFeeInSellToken ?? 0)?.toFixed(
-                      3
-                    )}{" "}
-                {symbol}
-              </>
-            ) : (
-              "Choose currency"
+          <Skeleton
+            isLoaded={Boolean(
+              pickedCurrency && calculatedGasFee && guildFeeInSellToken
```

Comment:
Is there no state when it appears loading because one of the conditions is falsy, but it's not actually loading just ran into an error?

Reply:
Good point, fixed!